scripts/check-devices.py

```diff
@@ -214,8 +222,7 @@ def cli_manager() -> argparse.Namespace:
     parser.add_argument(
         "-log",
         "--loglevel",
-        default="info",
-        help="Provide logging level. Example --loglevel debug, default=info",
+        help="Provide logging level. Example --loglevel debug, default=none",
```

Comment:
Maybe let's explain add the default behavior here as well

Reply:
Done with a short description to not make output not readable